services/QuillLMS/spec/controllers/concerns/teacher_fixes_spec.rb

```diff
@@ -451,4 +451,8 @@
     end
   end
 
+  describe '#recalculate_user_pack_sequence_items' do
+
+  end
```

Comment:
I think this was supposed to get deleted, since there's no method by that name.

Reply:
Good eye.  Yes, thank you!